4.0.0/migration.md

```diff
@@ -27,135 +146,148 @@ The migration process is initiated by the Installer if the following requirement
+        waitforpods.go:311] Waiting for deployment "sysdigcloud-collector" rollout to finish: 1 old replicas are pending termination...
+        waitforpods.go:311] Waiting for deployment "sysdigcloud-collector" rollout to finish: 1 old replicas are pending termination...
+        waitforpods.go:315] Waiting for deployment "sysdigcloud-api" rollout to finish: 0 of 1 updated replicas are available...
+        waitforpods.go:311] Waiting for deployment "sysdigcloud-collector" rollout to finish: 1 old replicas are pending termination...
+        waitforpods.go:315] Waiting for deployment "sysdigcloud-api" rollout to finish: 0 of 1 updated replicas are available...
+        waitforpods.go:311] Waiting for deployment "sysdigcloud-collector" rollout to finish: 1 old replicas are pending termination...
+        waitforpods.go:311] Waiting for deployment "sysdigcloud-collector" rollout to finish: 1 old replicas are pending termination...
+        waitforpods.go:315] Waiting for deployment "sysdigcloud-api" rollout to finish: 0 of 1 updated replicas are available...
+        waitforpods.go:318] deployment "sysdigcloud-collector" successfully rolled out
+        waitforpods.go:315] Waiting for deployment "sysdigcloud-api" rollout to finish: 0 of 1 updated replicas are available...
+        waitforpods.go:315] Waiting for deployment "sysdigcloud-api" rollout to finish: 0 of 1 updated replicas are available...
+        waitforpods.go:318] deployment "sysdigcloud-api" successfully rolled out
+        waitforpods.go:74] waitforpods(sysdigcloud-collector,sysdigcloud-api) completed in 1m32s
+
+
+8.  Ensure that collector block for the new agent connections is disabled (`GET /api/admin/agents/new/connections?block=false` HTTP request to API will be performed under the hood):
 
         deploy.go:1560] toggled collector block to false: Accepting new agent connections is ON
 
-9. Ensure that MySQL deployments (non-HA setup) or stateful sets (HA setup) and services are deleted.
+
+9. Ensure that MySQL deployments (non-HA setup) or stateful sets (HA setup) and services are deleted:
 
         deploy.go:1664] removing mysql deployments, stateful sets and services
 
-   At this point, the migration is successfully concluded and installation will continue as before.
-   For the following runs, installer will always generate the manifest for `mysql-to-postgresql-migration` and `mysql-latest-migrations jobs:`.
-   Even if you keep generating the manifests, the migration will not run again if it is once completed.
+   At this point, the migration is successfully concluded and installation will continue as usual.
+
 
 10.  As the PVC is retained by default, delete it after the migration is completed.
 
+
+**For the following runs, Installer will always generate the manifests for `mysql-to-postgresql-migration` and `mysql-latest-migrations` jobs: even if so, the migration will not run again if it has completed once.**
```

Comment:
**For the subsequent runs, Installer will always generate the manifests for `mysql-to-postgresql-migration` and `mysql-latest-migrations` jobs. The migration will not run again if it has completed once.**